crates/router/src/connector/bitpay.rs

```diff
@@ -27,8 +31,18 @@ use crate::{
     utils::BytesExt,
 };
 
-#[derive(Debug, Clone)]
-pub struct Bitpay;
+#[derive(Clone)]
+pub struct Bitpay {
+    amount_converter: &'static (dyn AmountConvertor<Output = MinorUnit> + Sync),
+}
+
+impl Bitpay {
+    pub fn new() -> &'static Self {
```

Comment:
Same here

Reply:
here also lifetime is added since struct requires `static` lifetime